go/libkb/pgp_key.go

```diff
@@ -255,6 +255,10 @@ func PGPKeyRawToArmored(raw []byte, priv bool) (ret string, err error) {
 	return
 }
 
+func (k *PGPKeyBundle) SerializePrivate(w io.Writer) error {
+	return k.Entity.SerializePrivate(w, &packet.Config{ReuseSignaturesOnSerialize: !k.Generated})
```

Comment:
new sigs are made.  which need to happen in the generated case.

we generate an encryption subkey but not signing subkey, so we get away with it :/